File: compsci/ios/tourObjectiveC/prj/objectivec/ch1-intro/TekPubMobileApp/TekPubMobileApp/ViewController.m
//
//  ViewController.m
//  TekPubMobileApp
//
//  Created by Sviatoslav Zimine on 10/26/17.
//  Copyright © 2017 Sviatoslav Zimine. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController
@synthesize welcomeLabel;

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}

-(void) viewDidUnload
{
    [self setWelcomeLabel:nil];
    [super viewDidUnload];
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
}


- (IBAction)onButtonTapped:(id)sender {
    welcomeLabel.text = [NSString stringWithFormat:@"Hello from TEKPub"];
    [UIView animateWithDuration:1.0 animations:^{     // this is a closure
        self.view.backgroundColor = [ UIColor redColor];
        welcomeLabel.frame = CGRectMake(0, 0, 200, 50);
    }];
}


- (void)dealloc {
    [welcomeLabel release];
    [super dealloc];
}
@end

File: compsci/ios/chat-cloudbased-firebase/ex/ch4_scene_layout/pchatapp/ViewController.swift
//
//  ViewController.swift
//  pchatapp
//
//  Created by Sviatoslav Zimine on 3/15/18.
//  Copyright © 2018 Sviatoslav Zimine. All rights reserved.
//

import UIKit

class ViewController: UIViewController {


    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    @IBAction func LoginButton_click(_ sender: Any) {
        self.performSegue(withIdentifier: "showProfile", sender: sender)
    }
    
}

